src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -78,7 +79,7 @@ public static String buildSearchConceptsQuery(
     List<FilterVariable> allFilters = List.of(domainClause, searchClause);
 
     FilterVariable where;
-    if (searchText.isEmpty()) {
+    if (StringUtils.isNullOrEmpty(searchText)) {
```

Comment:
The commons form of this is `isEmpty()` since that handles `null` as well.
```suggestion
    if (StringUtils.isEmpty(searchText)) {
```

Personally though I'm not a fan of using `StringUtils` for trivial cases like this, I would instead check the object directly.
```suggestion
    if (searchText == null || searchText.isEmpty()) {
```

but this is a matter of preference

Reply:
I've changed to `isEmpty()` because `isNullOrEmpty()` is not in `StringUtils`.